spec/plans/remote.fmf

```diff
@@ -0,0 +1,44 @@
+summary: Referencing remote plans
+
+story:
+    As a user I want to reference a plan from a remote repository
+    in order to prevent duplication and minimize maintenance.
+
+description: |
```

Comment:
@psss should we also describe a bit in detail how will a remote plan inclusion influence the plan where we do the include of the remote plan? Maybe with some examples? Or is that not decided yet?

Reply:
What do you mean by "influence"? The remote plan will be fetched and its steps will replace the `plan` key with the real step configuration. Everything should work completely the same.

Reply:
ok, I guess I got my answers on the call